package.json

```diff
@@ -3,29 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "node index.js",
-    "serve": "vue-cli-service serve",
+    "start": "nodemon index.js",
+    "serve": "PORT=8081 vue-cli-service serve",
+    "dev-start": "concurrently \"npm run start\" \"npm run serve\"",
```

Comment:
@jinalviranii dev-start command should support HMR on the same port on which backend server is running